ios/Events/CardVerificationChoiceSelectedObserver.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  CardVerificationChoiceSelectedObserver.swift
+//  fidel-react-native
+//
+//  Created by Valeria Rogatchevskikh on 8/6/23.
+//
+
+import Foundation
+
+import Foundation
+import Fidel
+
+struct CardVerificationChoiceSelectedObserver: EventObserver {
+    func startObserving(_ onEvent: @escaping (NSDictionary) -> Void) {
+        Fidel.onCardVerificationChoiceSelected = {
+            let dictionary: NSMutableDictionary = NSMutableDictionary()
+            switch($0) {
+            case .delegatedToThirdParty:
+                dictionary["verificationChoice"] = "ON_THE_SPOT"
```

Comment:
All the constants that we exposed, so far, use camel case (check the countries, for example). For consistency, I would suggest maintaining this convention.